yt_dlp/extractor/vtm.py

```diff
@@ -1,60 +1,42 @@
-from .common import InfoExtractor
-from ..utils import (
-    int_or_none,
-    parse_iso8601,
-    try_get,
-)
+from .medialaan import MedialaanBaseIE
+from ..utils import str_or_none
+from ..utils.traversal import require, traverse_obj
 
 
-class VTMIE(InfoExtractor):
-    _WORKING = False
-    _VALID_URL = r'https?://(?:www\.)?vtm\.be/([^/?&#]+)~v(?P<id>[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})'
-    _TEST = {
+class VTMIE(MedialaanBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?vtm\.be/[^/?#]+~v(?P<id>[\da-f-]+)'
+    _TESTS = [{
         'url': 'https://vtm.be/gast-vernielt-genkse-hotelkamer~ve7534523-279f-4b4d-a5c9-a33ffdbe23e1',
-        'md5': '37dca85fbc3a33f2de28ceb834b071f8',
         'info_dict': {
             'id': '192445',
             'ext': 'mp4',
             'title': 'Gast vernielt Genkse hotelkamer',
-            'timestamp': 1611060180,
-            'upload_date': '20210119',
+            'channel': 'VTM',
+            'channel_id': '867',
+            'description': 'md5:75fce957d219646ff1b65ba449ab97b5',
             'duration': 74,
-            # TODO: fix url _type result processing
-            # 'series': 'Op Interventie',
+            'genres': ['Documentaries'],
+            'release_date': '20210119',
+            'release_timestamp': 1611060180,
+            'series': 'Op Interventie',
+            'series_id': '2658',
+            'tags': 'count:2',
+            'thumbnail': r're:https?://images\.mychannels\.video/imgix/.+\.(?:jpe?g|png)',
+            'uploader': 'VTM',
+            'uploader_id': '74',
         },
-    }
+    }]
+
+    def _real_initialize(self):
+        if not self._get_cookies('https://vtm.be/').get('authId'):
+            self.raise_login_required()
 
     def _real_extract(self, url):
         uuid = self._match_id(url)
```

Comment:
let's avoid shadowing a stdlib module name
```suggestion
        video_id = self._match_id(url)
```